search_that_hash/cracker/sth_mod/sth.py

```diff
@@ -35,16 +38,16 @@ def crack(self, config):
 
         return output, config
 
-    def push(self, chash: str, result: str, types: list):
+    def push(self, chash: str, result: str, types: list):  # pragma: no cover
         url = "https://av5b81zg3k.execute-api.us-east-2.amazonaws.com/prod/insert"
         headers = {
             "x-api-key": f"{self.config['api_keys']['STH']}",
             "Content-Type": "application/json",
         }
         payload = json.dumps({"Hash": chash, "Plaintext": result, "Type": types})
         try:
-            requests.request("PUT", url, headers=headers, data=payload)
-        except Exception:

```

Comment:
Why did you get rid of `Exception`? It's better than having a bare try except.

Reply:
Yeah my bad. Fixed